### PR TITLE
mrc-2370 Create an endpoint for model calibrate bar chart

### PR DESCRIPTION
**Pull request by Nicolas-Dolan**
## Description

This endpoint calls the corresponding hintr endpoint [/calibrate/plot/{id}](https://github.com/mrc-ide/hintr/pull/274) (formerly `/calibrate/estimates/{id}`) for the data.

**Merging this PR is dependent on mrc-2398 being merged and `hintr_version` being reverted**

## Type of version change
Minor

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

**Review by mwoodbri — COMMENTED** [review]
Is the relevant hintr endpoint the one being implemented in mrc-2398? If so then that ticket should be specified as a dependency on the mrc-2370 ticket, the `hintr_version` file on this branch should point at that branch, and the `can get calibrate plot` test should be checking for something other than a `404`.

**Comment by Nicolas-Dolan** [comment]
> Is the relevant hintr endpoint the one being implemented in [mrc-2398](https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2398)? If so then that ticket should be specified as a dependency on the [mrc-2370](https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2370) ticket, the `hintr_version` file on this branch should point at that branch, and the `can get calibrate plot` test should be checking for something other than a `404`.

I've now updated the youtrack ticket and pointed the pr at mrc-2398

**Review by mwoodbri — APPROVED** [review]
Looks good, though merge blocked by mrc-2398.

Should note that changes dependent on this endpoint can't themselves be released until the mintr endpoint is fully implemented.

File: src/config/hintr_version
Comment (carbowizzy) [inline src/config/hintr_version:1]: I am not sure why this is not master. Does that mean it's not mergeable until the hintr branch is merged ?

**Review by carbowizzy — APPROVED** [review]

File: src/config/hintr_version
Comment (Nicolas-Dolan) [reply src/config/hintr_version:1]: Yes, that's correct.

**Comment by Nicolas-Dolan** [comment]
This was previously approved. Now that the hintr dummy endpoint has been merged in, this should be ready to merge.

**Review by mwoodbri — APPROVED** [review]

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**
